### DISCUSSION TITLE
Accounting

### DISCUSSION THREAD

**Discussion by sohrabp72** [freifunkMUC/wg-access-server#675] [General]
Does this panel support peer traffic limit or peer expiration?

**Comment by GoliathLabs** [comment]
We currently don't have such restrictions. That would require PR.

**Reply by sohrabp72** [reply]
sorry if I ask this newbie question what PR stands for?

**Reply by GoliathLabs** [reply]
A pull request. Somebody would have to add those features and submit them to the project.
